bftengine/src/bcstatetransfer/BCStateTran.cpp

```diff
@@ -308,10 +307,11 @@ void BCStateTran::init(uint64_t maxNumOfRequiredStoredCheckpoints,
     } else {
       LOG_INFO(STLogger, "initializing a new object");
 
-      AssertAND(maxNumOfRequiredStoredCheckpoints >= 2,
-                maxNumOfRequiredStoredCheckpoints <= kMaxNumOfStoredCheckpoints);
+      AssertGE(maxNumOfRequiredStoredCheckpoints, 2);
+      AssertLE(maxNumOfRequiredStoredCheckpoints, kMaxNumOfStoredCheckpoints);
+      AssertGE(numberOfRequiredReservedPages, 2);
+      AssertLE(numberOfRequiredReservedPages, config_.maxNumOfReservedPages);
```

Comment:
this assert didn't exist previously, right?

Reply:
It did. There's an `AssertAND` that was replaced above and below these 4 asserts. 2 Asserts for each AssertAnd.

Reply:
Yep, you're right, thanks for the clarification!